tests/phpunit/tests/rest-api/rest-application-passwords-controller.php

```diff
@@ -549,21 +550,20 @@ public function test_update_item_cannot_overwrite_app_id() {
 		$response = rest_do_request( $request );
 		$this->assertSame( '', $response->get_data()['app_id'] );
 
-		$app_id = wp_generate_uuid4();
-
-		list( , $item ) = WP_Application_Passwords::create_new_application_password(
+		$error = WP_Application_Passwords::create_new_application_password(
 			self::$admin,
 			array(
 				'name'   => 'App',
-				'app_id' => $app_id,
+				'app_id' => wp_generate_uuid4(),
 			)
 		);
+		$this->assertInstanceOf( WP_Error::class, $error );
```

Comment:
I don't think this is correct, it looks like it is testing something different now. Instead, the `name` should be updated for this second password creation.

Reply:
My misunderstanding. Thank you for flagging it.

Reverted the test and then changed the App name in commit https://github.com/WordPress/wordpress-develop/pull/898/commits/95260a0e36652960fe7510f4ba4a7aa394873966.

Reply:
@TimothyBJacobs Test name implies "update".  Shouldn't this test be using `::update_application_password` instead of `::create_new_application_password` for the "update"?

Reply:
No, the update refers to the `PUT` request. We're creating the app password fixture using `create_new_application_password`.